app/mailers/audit_objects_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# Send report of invalid objects to admin team
+class AuditObjectsMailer < ActionMailer::Base
```

Comment:
Thanks for the thorough review.

Nice to know about the preview mechanism (my previous job was Rails 3.2, and this functionality was available through a separate gem/engine). Also at the previous job, we used a markdown gem for rendering multi-part emails (the text version would just be the markdown, which is for the most part readable as text).